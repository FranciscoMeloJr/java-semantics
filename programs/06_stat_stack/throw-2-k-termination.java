//Throw causes thread termination.

class Exception {
  Exception(){}
}

class main {
  void main(String[] args) {
    throw new Exception();
    System.out.print("unreachable");
  }
}
// Thread terminated with exception: Exception
