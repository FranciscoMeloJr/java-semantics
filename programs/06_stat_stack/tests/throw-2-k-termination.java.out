Thread terminated with exception: Exception
